### PR TITLE
Charlesmchen/app extension capabilities and info

### PR DESCRIPTION
**Pull request by charlesmchen**
PTAL @michaelkirk

### CODE REVIEW

File: Signal.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -3121,13 +3138,13 @@
 				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
```
Comment (charlesmchen) [inline Signal.xcodeproj/project.pbxproj]: I didn't modify this line, but it's notable: it appears we can target a different minimum version of iOS for our share extension. Score.

File: Signal.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -3121,13 +3138,13 @@
...
 				MTL_ENABLE_DEBUG_INFO = YES;
-				PRODUCT_BUNDLE_IDENTIFIER = org.whispersystems.signal.SignalShareExtension;
+				PRODUCT_BUNDLE_IDENTIFIER = org.whispersystems.signal.shareextension;
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SKIP_INSTALL = YES;
 				SWIFT_ACTIVE_COMPILATION_CONDITIONS = DEBUG;
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
 				SWIFT_VERSION = 4.0;
-				TARGETED_DEVICE_FAMILY = "1,2";
+				TARGETED_DEVICE_FAMILY = 1;
```
Comment (charlesmchen) [inline Signal.xcodeproj/project.pbxproj]: Phone only?

File: SignalShareExtension/Info.plist [hunk 3]
```diff
@@ -17,9 +17,9 @@
 	<key>CFBundlePackageType</key>
 	<string>XPC!</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.0</string>
+	<string>2.20.0</string>
 	<key>CFBundleVersion</key>
-	<string>1</string>
+	<string>2.20.0.0</string>
```
Comment (michaelkirk) [inline SignalShareExtension/Info.plist]: For now we're keeping this in lock-step with the main app manually?

Comment (charlesmchen) [reply SignalShareExtension/Info.plist]: See https://github.com/WhisperSystems/Signal-iOS/pull/2831

I'm eventually also going to add a command line switch to change the release number.

**Pull request merged**